includes/Services/PluginService.php

```diff
@@ -41,6 +64,9 @@ public static function initialize() {
 			$init_plugins = array_merge( Plugins::get_init(), SiteFeatures::get_init() );
 		}
 
+		// Add plugins enabled by Hiive Flags
+		$init_plugins = array_merge( $init_plugins, self::initialize_hiive_flag_plugins() );
```

Comment:
Do we still need to perform the plan checklist at https://github.com/newfold-labs/wp-module-onboarding-data/blob/a5e90dbdb748e2710f82654131b51973a6dabd12/includes/Plugins.php#L25? Is it necessary to verify the plan at https://github.com/newfold-labs/wp-module-onboarding-data/blob/a5e90dbdb748e2710f82654131b51973a6dabd12/includes/Plugins.php#L213? Can we simply load the plugins based on the capability? I'm just wondering if we can take this opportunity to simplify all of this complex logic.

Reply:
I will be updating this once I receive confirmation from the other end.

Reply:
Done